### PR TITLE
fix #8799 - typo in EXISTING_USER_UNSUPPORTED_WINDOWS_VERSION …

### LINKED ISSUE #8799: Typo in EXISTING_USER_UNSUPPORTED_WINDOWS_VERSION slug
slug="existing_user_**usupported**_windows_version" when it should be unsupported 😅

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3465)

### PR DESCRIPTION
**Pull request by nuggetsnegin**
Because 
* There's a typo `EXISTING_USER_UNSUPPORTED_WINDOWS_VERSION` targeting slug (`usupported` vs `unsupported`)

This commit
* Fixes the typo I introduced 😅

### CODE REVIEW

**Review by aminomancer — APPROVED** [review]
LGTM, r+

To be clear, this config has yet to be used by any experiments. Negin and Hanna have 2 upcoming experiments that will use it, but will hold off on defining the experiments until this lands and the change is reflected on Experimenter.

I think @jaredlockhart would have a better sense of whether changing the slug causes any other problems. Thanks!

**Review by yashikakhurana — APPROVED** [review]

**Comment by jaredlockhart** [comment]
@aminomancer Yes exactly, as long as the slug isn't used anywhere we shouldn't have a problem.  Thanks for the update!

**Pull request merged**
